src/main/java/frc/robot/commands/AcquireCommand.java

```diff
@@ -1,31 +1,47 @@
 package frc.robot.commands;
 
+import edu.wpi.first.wpilibj.Timer;
 import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.AcquisitionConstants;
 import frc.robot.subsystems.Acquisition;
 
 public class AcquireCommand extends Command {
     private static Acquisition acquisition = Acquisition.getInstance();
+    private Timer timer = new Timer();
+    private double timeToAcquire;
 
-    private double speed;
+    public AcquireCommand() {
+        addRequirements(acquisition);
+        this.timeToAcquire = AcquisitionConstants.defaultTimeToAcquire;
+    }
 
-    public AcquireCommand(double speed) {
+    public AcquireCommand(double timeToAcquire) {
         addRequirements(acquisition);
-        this.speed = speed;
+
+        this.timeToAcquire = timeToAcquire;
+    }
+
+    @Override
+    public void initialize() {
+        timer.start();
     }
 
     @Override
     public void execute() {
-        acquisition.acquire(speed);
+        acquisition.acquire();
     }
 
     @Override
     public boolean isFinished() {
         // TODO: update with sensors
-        return false;
+
+        return acquisition.isNotePresent() || (timeToAcquire == 0) || (timer.get() > timeToAcquire);
```

Comment:
The note is not acquired until the laser sees the note and then stops seeing the note. In other words, the laser will see the top edge of the note as it is being pulled in, but the note is not "acquired" until after the far edge is past the laser